### ISSUE TITLE
Try to run on Jetson AGX Orin

### ISSUE THREAD

**Issue by mymistakes** [KoljaB/RealtimeSTT#275]
`Package            Version                   Editable project location
------------------ ------------------------- -------------------------------------------
av                 15.0.0
certifi            2025.7.14
cffi               1.17.1
charset-normalizer 3.4.2
colorama           0.4.6
coloredlogs        15.0.1
ctranslate2        4.6.0
enum34             1.1.10
faster-whisper     1.1.1
filelock           3.18.0
flatbuffers        25.2.10
fsspec             2025.7.0
halo               0.0.31
hf-xet             1.1.5
huggingface-hub    0.34.3
humanfriendly      10.0
idna               3.10
Jinja2             3.1.6
joblib             1.5.1
log-symbols        0.0.14
MarkupSafe         3.0.2
mpmath             1.3.0
networkx           3.4.2
numpy              1.26.4
onnxruntime        1.22.1
openwakeword       0.6.0
packaging          25.0
pip                22.0.2
protobuf           6.31.1
pvporcupine        1.9.5
PyAudio            0.2.14
pycparser          2.22
PyYAML             6.0.2
realtimestt        0.3.104
requests           2.32.4
scikit-learn       1.7.1
scipy              1.15.2
setuptools         59.6.0
six                1.17.0
soundfile          0.13.1
spinners           0.0.24
sympy              1.13.1
termcolor          3.1.0
tflite-runtime     2.14.0
threadpoolctl      3.6.0
tokenizers         0.21.4
torch              2.5.0a0+872d972e41.nv24.8
torchaudio         2.5.0                     /pytorch-tools/audio/src
tqdm               4.67.1
typing_extensions  4.14.1
urllib3            2.5.0
webrtcvad-wheels   2.0.14
websocket-client   1.8.0
websockets         15.0.1`

The above is the Python environment for my Jetson AGX Orin. When I run "from RealtimeSTT import AudioToTextRecorder", the following error message appears:
`Python 3.10.12 (main, May 27 2025, 17:12:29) [GCC 11.4.0] on Linux
Type "help", "copyright", "credits" or "license" for more information.
>>> from RealtimeSTT import AudioToTextRecorder
/opt/rh/gcc-toolset-14/root/usr/include/c++/14/bits/stl_vector.h:1130: std::vector<_Tp, _Alloc>::reference std::vector<_Tp, _Alloc>::operator[](size_type) [with _Tp = unsigned int; _Alloc = std::allocator<unsigned int>; reference = [unsigned int&; size_type = long unsigned int]: Assertion '__n < this->size()' failed.

Aborted`

Does anyone know how to fix this?

**Comment by mymistakes** [comment]
OK, this problem is caused by the version of onnxruntime. I installed the version [](https://github.com/ultralytics/assets/releases/download/v0.0.0/onnxruntime_gpu-1.20.0-cp310-cp310-linux_aarch64.wh) and then encountered the problem that pvporcupine does not support Jetson. I commented out "import pvporcupine" in the source code and now it can run.

**Comment by mymistakes** [comment]
OK, this problem is caused by the version of onnxruntime. I installed the version https://github.com/ultralytics/assets/releases/download/v0.0.0/onnxruntime_gpu-1.20.0-cp310-cp310-linux_aarch64.whl, and then encountered the problem that pvporcupine does not support Jetson. I commented out "import pvporcupine" in the source code and now it can run.
